lochness/mediaflux/__init__.py

```diff
@@ -74,7 +74,11 @@ def sync_module(Lochness: 'lochness.config',
                       product: GENEActiv
                       data_dir: all_BWH_actigraphy
```

Comment:
If PRESCIENT is going to follow ProNET, shouldn't `all_BWH_actigraphy` be changed to `PrescientXX_Actigraphy`?

Reply:
Correct

Reply:
5700f6b

Reply:
The commit needs correction.